README.md

```diff
@@ -123,11 +128,20 @@ Forgeries is available through [CocoaPods](http://cocoapods.org). To install it,
 target 'MyApp_Tests' do
   inherit! :search_paths
 
-  pod 'forgeries'
+  pod 'Forgeries'
   ...
 end
 ```
 
+That will import the core functionality, which is currently just `UIGestureRecognizer` subclasses. For everything else, use:
```

Comment:
Not really sure I'm down with removing `NSUserDefaults` / `NSFileManager` from the default, they both only have one function that uses OCMock.